graph/layout/eades_example_test.go

```diff
@@ -0,0 +1,198 @@
+// Copyright ©2021 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file
+
+package layout_test
+
+import (
+	"fmt"
+	"image/color"
+	"log"
+	"math"
+	"path/filepath"
+
+	"gonum.org/v1/gonum/graph/layout"
+	"gonum.org/v1/gonum/graph/simple"
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+	"gonum.org/v1/plot/vg/draw"
+)
+
+func ExampleEadesR2() {
+	// Make a simple graph and render it as a PNG
+	// with the EadesR2 force-directed layout.
+	g := makeCompleteGraph(6)
+
+	// Graph layout.
+	// Explained in "A heuristic for graph drawing",
+	// Congressus numerantium 42:149-160.
+	// Experiment with these values with more/less nodes.
```

Comment:
Replace with

```
// Use the Eades layout algorithm with reasonable defaults.
```

(There's already documentation attached so the repetition is not needed).